catalog-info.yaml

```diff
@@ -0,0 +1,19 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'edx-rest-api-client'
+  description: "Utility library to simplify REST API access"
+  annotations:
+    # (Optional) Annotation keys and values can be whatever you want.
+    # We use it in Open edX repos to have a comma-separated list of GitHub user
+    # names that might be interested in changes to the architecture of this
+    # component.
+    openedx.org/arch-interest-groups: ""
+spec:
+  # (Required) This can be a group(`group:<group_name>` or a user(`user:<github_username>`)
+  owner: user:nedbat
```

Comment:
Is this temporary? Or just part of your question on how to list an individual and a team? This is owned by arch-bom, right?

Reply:
I've changed the entry to `owner: group:openedx/arch-bom`, though I still have an open question about when we want to talk about teams and when we want to talk about individuals.

@feanil: note that groups should include the GitHub organization, and a few of our existing catalog-info.yaml files don't include the "group:" prefix:
```
openedx/openedx-events/catalog-info.yaml
13:  owner: group:hooks-extension-framework

openedx/DoneXBlock/catalog-info.yaml
14:  owner: tcril-engineering

openedx/openedx-backstage/catalog-info.yaml
12:  owner: backstage-maintainers

openedx/openedx-translations/catalog-info.yaml
14:  owner: tcril-engineering

openedx/openedx-learning/catalog-info.yaml
8:  owner: tcril-engineering
```

Reply:
Further discussion (https://github.com/openedx/open-edx-proposals/pull/378) seems to indicate that the prefix is not needed, so I'm going with that for now.